src/si/momentum.rs

```diff
@@ -0,0 +1,201 @@
+//! Momentum (base unit kilogram meter per second, kg<sup>1</sup> · m<sup>1</sup> · s<sup>-1</sup>).
+
+quantity! {
+    /// Momentum (base unit kilogram meter per second, kg<sup>1</sup> · m<sup>1</sup> · s<sup>-1</sup>).
+    quantity: Momentum; "momentum";
+    /// Momentum dimension, kg<sup>1</sup> · m<sup>1</sup> · s<sup>-1</sup>).
```

Comment:
Remove the extra parenthesis at the end of the line.

Reply:
done